tests/foreman/cli/test_satellitesync.py

```diff
@@ -1139,6 +1139,91 @@ def test_positive_export_import_redhat_cv_with_huge_contents(
         for item in ['packages', 'source-rpms', 'package-groups', 'errata', 'module-streams']:
             assert exported_repo['content-counts'][item] == imported_repo['content-counts'][item]
 
+    @pytest.mark.tier2
+    def test_positive_export_cv_with_on_demand_repo(
```

Comment:
Non-blocking, I suggest combining positive/negative tests for exporting CVs as I see only assertion difference in the last part.

Reply:
The main difference is the _--fail-on-missing-content_ parameter. It was the reason why I split it into two tests.